test/unit/org/apache/cassandra/cql3/ast/CreateIndexDDL.java

```diff
@@ -134,6 +148,15 @@ else if (((isFrozen(type) && !type.isVector()) || StorageAttachedIndex.SUPPORTED
                 return true;
             return false;
         }
+
+        @Override
+        public EnumSet<QueryType> supportedQueries(AbstractType<?> type)
+        {
+            type = type.unwrap();
```

Comment:
`Symbol.type` is never reversed, you need `Symbol.rawType()` to get the reversed type... but since this works with types directly we have to be defensive off the bat...